MyBlog.Services/Areas/Users/UserService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.EntityFrameworkCore;
+using MyBlog.Infrastructure;
+using MyBlog.Infrastructure.Entities;
+using MyBlog.Service.Exception;
+
+namespace MyBlog.Service.Areas.Users;
+
+public class UserService : IUserService
+{
+    private readonly MyBlogContext _context;
+
+    public UserService(MyBlogContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<List<User>> GetListAsync()
+    {
+        var userList = await _context.Users.ToListAsync();
```

Comment:
if you use method ToList async it`s easy to understand, that it will be userList,
so you can name this variable just users.